### PR TITLE
CB-18696 some creator field used in responses, cannot be removed but …

### PR DESCRIPTION
**Pull request by horadla23**
…can be marked as deprecated

See detailed description in the commit message.

### CODE REVIEW

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/base/ImageCatalogV4Base.java [hunk 1]
```diff
@@ -29,13 +29,28 @@ public class ImageCatalogV4Base {
...
+     * and can become invalid, usage of it can be error prune
+     */
+    @Deprecated
     public String getCreator() {
         return creator;
     }
 
+    /**
+     * @deprecated data owner of any user is UMS, creator should not be stored and used anywhere, since user of creator can leave the given company
+     * and can become invalid, usage of it can be error prune
```
Comment (lturcsanyi) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/base/ImageCatalogV4Base.java:51]: prune -> prone

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/base/ImageCatalogV4Base.java [hunk 2]
```diff
@@ -29,13 +29,28 @@ public class ImageCatalogV4Base {
     @ValidImageCatalog
     private String url;
 
+    /**
+     * @deprecated data owner of any user is UMS, creator should not be stored and used anywhere, since user of creator can leave the given company
+     * and can become invalid, usage of it can be error prune
+     */
+    @Deprecated
     @ApiModelProperty(ModelDescriptions.CREATOR)
```
Comment (lturcsanyi) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/base/ImageCatalogV4Base.java:37]: Could you please include it in the @ApiModelPropery descriptions (all occurrences) as well?

Comment (horadla23) [reply core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/base/ImageCatalogV4Base.java:37]: sure

File: core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java [hunk 3]
```diff
@@ -15,7 +15,9 @@ public class ModelDescriptions {
     public static final String CREATED = "creation time of the resource in long";
     public static final String AMBARI_SERVER = "ambari server address";
     public static final String WORKSPACE_OF_THE_RESOURCE = "workspace of the resource";
-    public static final String CREATOR = "the creator of the resource";
+    public static final String CREATOR = "the creator of the resource - Deprecated: data owner of any user is UMS, " +
+            "creator should not be stored and used anywhere, since user of creator can leave the given company and can become invalid, " +
+            "usage of it can be error prune";
```
Comment (lturcsanyi) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java:20]: prune -> prone :)

File: environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/model/response/CredentialResponse.java [hunk 4]
```diff
@@ -33,6 +33,11 @@ public class CredentialResponse extends CredentialBase {
     @ApiModelProperty(CRN)
     private String crn;
 
+    /**
+     * @deprecated data owner of any user is UMS, creator should not be stored and used anywhere, since user of creator can leave the given company
+     * and can become invalid, usage of it can be error prone
+     */
+    @Deprecated
     @ApiModelProperty(CredentialModelDescription.CREATOR)
```
Comment (lturcsanyi) [inline environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/model/response/CredentialResponse.java:41]: Please include into `CredentialModelDescription.CREATOR` as well.

File: environment-api/src/main/java/com/sequenceiq/environment/api/v1/proxy/model/response/ProxyResponse.java [hunk 5]
```diff
@@ -22,6 +22,11 @@ public class ProxyResponse extends ProxyBase {
     @ApiModelProperty(ProxyConfigDescription.PASSWORD)
     private SecretResponse password;
 
+    /**
+     * @deprecated data owner of any user is UMS, creator should not be stored and used anywhere, since user of creator can leave the given company
+     * and can become invalid, usage of it can be error prone
+     */
+    @Deprecated
     @ApiModelProperty(ProxyConfigDescription.CREATOR)
```
Comment (lturcsanyi) [inline environment-api/src/main/java/com/sequenceiq/environment/api/v1/proxy/model/response/ProxyResponse.java:30]: Please include into `ProxyConfigDescription.CREATOR` as well.

File: core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java (hunk 3 above)
Comment (horadla23) [reply core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java:20]: ahh, again :D

**Review by lturcsanyi — APPROVED** [review]

**Pull request merged**